docs/csharp/linq/perform-inner-joins.md

```diff
@@ -65,6 +77,14 @@ The result of `query1` is equivalent to the result set that would have been obta
 
 :::code language="csharp" source="../../../samples/snippets/csharp/concepts/linq/LinqSamples/InnerJoins.cs" id="inner_joins_4":::
 
+The same results can be achieved using <xref:System.Linq.Enumerable.GroupJoin%2A> method, as follows:
+
+:::code language="csharp" source="../../../samples/snippets/csharp/concepts/linq/LinqSamples/InnerJoins.cs" id="inner_joins_method_syntax_4":::
+
+note that this approach requires chaining the query results with <xref:System.Linq.Enumerable.SelectMany%2A> which creates the final list of Owner - Pet relation based on the results of group join. To avoid chaining, the single <xref:System.Linq.Enumerable.Join%2A> method can be used as presented below:
```

Comment:
capitalization:

```suggestion
Note that this approach requires chaining the query results with <xref:System.Linq.Enumerable.SelectMany%2A> to create the final list of Owner - Pet relation based on the results of group join. To avoid chaining, the single <xref:System.Linq.Enumerable.Join%2A> method can be used as presented below:
```